project-management/Github.md

```diff
@@ -0,0 +1,77 @@
+# Managing public (or private) projects using GitHub
+
+We use a common set of labels across projects in GitHub. It is
+possible to precisely control this set of labels using
+the [settings GitHub app][github-app-settings] (needs to be enabled
+for your org). Use the [`settings.yml`](../.github/settings.yml) from
+this repository as a starting point.
+
+[github-app-settings]: https://github.com/probot/settings
+
+## Labels
+
+We use labels to classify tasks by priority, from P0 to P4:
+
+* **P0 (blocker)** Do not do anything else, this is a blocker.
+* **P1 (critical)** Must have.
+* **P2 (major)** Should have.
+* **P3 (minor)** Could have.
+* **P4 (trivial)** Someday/maybe. Consider closing as "wontfix".
+
+Labels are also useful to classify issues by type or by component. Use
+colours to distinguish whether a label designates a type, a component
+or something else.
```

Comment:
There seem to be lacking a discussion about github projects and boards. (whether to use them and what the workflow would be).